File: app/src/main/java/com/example/whattowatch/ui/presenter/MainPresenter.java
package com.example.whattowatch.ui.presenter;

import android.util.Log;

import com.arellomobile.mvp.InjectViewState;
import com.arellomobile.mvp.MvpPresenter;
import com.example.whattowatch.ui.view.MainView;

@InjectViewState
public class MainPresenter extends MvpPresenter<MainView> {

    public MainPresenter() {
    }

    @Override
    protected void onFirstViewAttach() {
        super.onFirstViewAttach();
        getViewState().showPagerView();// при создании отображает основной экрн
    }


}

File: app/src/main/java/com/example/whattowatch/ui/presenter/MainPresenter.kt
package com.example.whattowatch.ui.presenter

import com.arellomobile.mvp.InjectViewState
import com.arellomobile.mvp.MvpPresenter
import com.example.whattowatch.ui.view.MainView

@InjectViewState
class MainPresenter : MvpPresenter<MainView?>() {
    override fun onFirstViewAttach() {
        super.onFirstViewAttach()
        viewState!!.showPagerView() // при создании отображает основной экрн
    }
}